bin/disco_rds.py

```diff
@@ -55,6 +56,16 @@ def get_parser():
     parser_cleanup_snapshots.set_defaults(mode="cleanup_snapshots")
     parser_cleanup_snapshots.add_argument('--age', dest='days', required=False,
                                           help='Minimum age of Snapshots to expire', type=int, default=30)
+
+    # clone Mode
+    parser_clone = subparsers.add_parser("clone", help="Create a new database from an existing database")
+    parser_clone.set_defaults(mode="clone")
+    parser_clone.add_argument('--env', dest='env', required=False, default=None,
+                              help='The environment containing the RDS cluster(s)')
+    parser_clone.add_argument('--source-db', dest='source_db', required=True,
```

Comment:
This argument leaves me thinking there will be a "destination-db" as well, which is of course untrue.  Would just using "--cluster", the same way create/delete do, make the clarity of the situation better or worse?  I kind of like it for consistency, but I may be missing an important technicality or five.